src/cf-layout/src/cf-layout.less

```diff
@@ -650,36 +650,35 @@
     });
   }
 }
-//
-// Hero
-//
+
 
 @hero-desktop-height: 285px;
 @hero-image-height: 195px;
-@desktop-min: 801px;
+@hero-bg: #f7f8f9;
+@hero-overlay-bg: #5a5d61;
+@hero-overlay-text: white;
 
 .hero {
-    background-color: @block__bg;
+    background-color: @hero-bg;
 
-    &_wrapper {
-        box-sizing: border-box;
-        padding: unit(@grid_gutter-width / @base-font-size-px, em)
-                 unit(15px / @base-font-size-px, em);
+    .respond-to-min( @bp-sm-min, {
+        display: table;
```

Comment:
I'm not seeing this problem locally. Are you sure that spacing under the text isn't making it slightly shorter than the image? I see that here:

![screen shot 2016-04-04 at 3 49 17 pm](https://cloud.githubusercontent.com/assets/1280430/14262795/e2531c5e-fa7c-11e5-9dba-3c29eb65c52f.png)

But it's not due to `table` and `table-cell`